### ISSUE TITLE
What about using libyui? TUI+GUI support in one!

### ISSUE THREAD

**Issue by timofonic** [linuxenko/chkservice#1]
Hello.

Thanks for your project, it seems quite interesting.

I use console most of the time, but sometimes I prefer the GUI approach. I often use remote shells and in those cases, I use console apps and many of them use ncurses/s-lang or whatever (as you may know, some programs use minimal parts of ncurses, but that's another topic).

What about using [libyui](https://github.com/libyui/)? This way, your program will be available for TUI (ncurses) and GUI (Gtk, Qt). it would make your program a lot more flexible and adapt better to different Desktop Environments. Originally it was developed for OpenSUSE's YaST, but it can be used by any project ;)

Kind regards.

**Comment by linuxenko** [comment]
@timofonic :unicorn: 

Thank you for response, i'll check the `libyui`, thank you. 
Anyways i'd like keep `chkservice` with no depndencies as it currently is.

**Comment by bl33pbl0p** [comment]
@linuxenko Adding a build time switch to not build with libyui is always an option, chkservice can finally be that one systemd Manager everyone has been looking for, if you integrate it with a GUI toolkit.

**Comment by linuxenko** [comment]
I don't think `chkservice` need any dependencies, it is small (about 30kb) tool that works just fine with minimal set of libraries.  The only UI thing is used here is main loop and keys , i don't use any panels, menus and any kind of ui things.. i don't use any of ui widgets from ncurses or libyui provide.

**Comment by bl33pbl0p** [comment]
Fair enough I guess, one can always fork and extend upon it if required, I do agree with keeping it simple.
